packages/lit-element/src/polyfill-support.ts

```diff
@@ -81,6 +81,8 @@ const polyfillSupport = ({LitElement}: {LitElement: PatchableLitElement}) => {
   };
 };
 
+typeof window === 'undefined' || (window['globalThis'] = window); // fix for iOS 11
```

Comment:
This seems reasonable, but let's put it in reactive-element's polyfill-support instead, as that also uses globalThis, and should run first

Reply:
Also, let's update the comment to mention that it's for older browsers in general